src/Cafe/OS/libs/nsyshid/Dimensions.cpp

```diff
@@ -547,13 +549,13 @@ namespace nsyshid
 		case 0xB1: // Seed
 		{
 			// Initialise a random number generator using the seed provided
-			g_dimensionstoypad.GenerateRandomNumber(&buf[4], sequence, q_result);
+			g_dimensionstoypad.GenerateRandomNumber(std::span<const uint8, 8>{buf.data() + 4, 8}, sequence, q_result);
```

Comment:
``data()`` should be changed to ``begin()``. They are subtly different. The std::span constructor expects an iterator type. data() returns a pointer type. In practice these are often equivalent, hence why it accepts the pointer as an argument, but this is not guaranteed anywhere. There are other places where you construct a span using a pointer and it should be changed there too.